Api/MidpagePaymentDataInterface.php

```diff
@@ -0,0 +1,32 @@
+<?php
+namespace Billink\Billink\Api;
+
+interface MidpagePaymentDataInterface
+{
+    /**
+     * @param int $cartId
+     * @param \Magento\Quote\Api\Data\PaymentInterface $paymentMethod
+     * @param \Magento\Quote\Api\Data\AddressInterface|null $billingAddress
+     * @return \Billink\Billink\Api\MidpageResultDataInterface $object
+     */
+    public function savePaymentInformationAndPlaceOrder(
+        $cartId,
```

Comment:
type of cartId

Depending on PHP/M2 version requirements, we can typehint the returntype too